src/System.CommandLine/ArgumentExtensions.cs

```diff
@@ -48,6 +48,13 @@ public static class ArgumentExtensions
             return argument;
         }
 
+        /// <summary>
+        /// Configures an argument to accept only the specified values, and to suggest them as command line completions.
```

Comment:
```suggestion
        /// Configures an argument to only accept the specified values, and to suggest them as command line completions.
```

Reply:
I shall not accept your split infinitive, sir!